### PR TITLE
test: reissue credential

### PR DESCRIPTION
**Pull request by sjoerdlmkns**

### CODE REVIEW

File: integration_test/issuance_test.dart [hunk 1]
```diff
@@ -21,89 +19,141 @@ void main() {
...
+
+    //   // Verify that the email address has not been added.
+    //   final emailTileFinder = find.byKey(const Key('irma-demo.sidn-pbdf.email_tile'));
+    //   expect(emailTileFinder, findsNothing);
+
+    //   // Verify that no activity has been added.
+    //   await tester.tap(find.byKey(const Key('nav_button_activity')));
+    //   await tester.pump(const Duration(seconds: 1));
+    //   expect(find.text('There are no logged activities yet'), findsOneWidget);
+    // });
```
Comment (ivard) [inline integration_test/issuance_test.dart:105]: Can these functions be deleted?

File: integration_test/issuance_test.dart [hunk 2]
```diff
@@ -21,89 +19,141 @@ void main() {
...
+      await tester.tapAndSettle(emailTileFinder);
+      expect(find.byType(CredentialsDetailScreen), findsOneWidget);
+      expect(credentialCardsFinder, findsOneWidget);
 
-      // Verify that no activity has been added.
-      await tester.tap(find.byKey(const Key('nav_button_activity')));
-      await tester.pump(const Duration(seconds: 1));
-      expect(find.text('There are no logged activities yet'), findsOneWidget);
+      // Expect the card to have the correct content again
+      await evaluateIssuedEmailAddressCard();
```
Comment (ivard) [inline integration_test/issuance_test.dart:156]: I think it's also nice to check whether the new credential instance actually replaced the old credential instance on the data tab. There should not be two credentials on the data tab with `test@example.com`.

File: integration_test/issuance_test.dart (hunk 1 above)
Comment (sjoerdlmkns) [reply integration_test/issuance_test.dart]: Oops 🤦,  i did not mean to stage this

File: integration_test/issuance_test.dart (hunk 2 above)
Comment (sjoerdlmkns) [reply integration_test/issuance_test.dart:158]: I do check this on the line above: `expect(credentialCardsFinder, findsOneWidget)` So there should only be one `IrmaCredentialCard` on the `CredentialsDetailScreen` after reissuing

Comment (ivard) [reply integration_test/issuance_test.dart:158]: Ah okay, I didn't notice that the data tab is still active after the second session.

**Review by ivard — APPROVED** [review]

**Review by MartijnYivi — APPROVED** [review]

**Pull request merged**